gems/pending/disk/modules/RawDiskProbe.rb

```diff
@@ -1,7 +1,7 @@
 module RawDiskProbe
   def self.probe(ostruct)
     return nil unless ostruct.fileName
-    return("RawDisk") if ostruct.rawDisk || File.extname(ostruct.fileName).downcase == ".img"
+    return("RawDisk") if ostruct.rawDisk
```

Comment:
Why did you remove the `or` condition? Do we encounter `.img` files that aren't raw?